{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol\n\nError: Source \"@cartesi/util/contracts/InstantiatorImpl.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:26:1:\n   |\n26 | import \"@cartesi/util/contracts/InstantiatorImpl.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@cartesi/util/contracts/Decorated.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:27:1:\n   |\n27 | import \"@cartesi/util/contracts/Decorated.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/ff/MMInterface.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:28:1:\n   |\n28 | import \"./MMInterface.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@cartesi/util/contracts/Merkle.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:29:1:\n   |\n29 | import \"@cartesi/util/contracts/Merkle.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/ff/Instantiator.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:509:1:\n    |\n509 | import \"./Instantiator.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@cartesi/util/contracts/Instantiator.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/ff/fF83d3A6eDe3664A180DdB7AabF05EAd15E64F4B_MMInstantiator.sol:705:1:\n    |\n705 | import \"@cartesi/util/contracts/Instantiator.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
